src/components/organisms/Nav.js

```diff
@@ -21,7 +25,13 @@ const Nav = () => {
   } = useAuthContext();
   const [modalState, setModalState] = useState(modalStates.closed);
   const [isAddingStory, setIsAddingStory] = useState(false);
-
+  const [isDisabled] = useState(true);
+  const [isUsername, setUsername] = useState('');
+  const [isEmail, setEmail] = useState('');
+  const [isPassword, setPassword] = useState('');
```

Comment:
These are not booleans, so you shouldn't name them as if they were "isBlahBlah". should just be "blahBlah"
```suggestion
  const [username, setUsername] = useState('');
  const [email, setEmail] = useState('');
  const [password, setPassword] = useState('');
```